### ISSUE TITLE
Bad test in `Batched3DModel3DTileContentSpec.js`

### ISSUE THREAD

**Issue by mramato** [AnalyticalGraphicsInc/cesium#4803]
Unless I'm missing something, `logs deprecation warning for use of BATCHID without prefixed underscore` is a bad test.  Here's the full test:

```javascript
it('logs deprecation warning for use of BATCHID without prefixed underscore', function() {
    var deprecationWarningSpy = jasmine.createSpy(deprecationWarning);
    return Cesium3DTilesTester.loadTileset(scene, withBatchTableUrl).then(function(tileset) {
        expect(deprecationWarningSpy).toHaveBeenCalled();
        Cesium3DTilesTester.expectRenderTileset(scene, tileset);
    });
});
```

`expect(deprecationWarningSpy).toHaveBeenCalled();` will always fail because you can't create a spy on a freestanding function like this.  Basically `var deprecationWarningSpy = jasmine.createSpy(deprecationWarning);` is a no-op since `deprecationWarningSpy` is never used.  When the internal Cesium code is ran, it always calls the real `deprecationWarning`.  If you want to spy on deprecation warning, you need to create a private inner function like we do with `loadWithXhr`.

I'm not sure how this code got into 3d-tiles since the test never passes when ran locally (it doesn't run on the server because it's WebGL).

CC @lilleyse @pjcozzi

**Comment by pjcozzi** [comment]
@lasalvavida can you look at this?  It should be an easy fix.

**Comment by lasalvavida** [comment]
Looking at this now

**Comment by pjcozzi** [comment]
Fixed in #4809
